src/query/plan/operator.cpp

```diff
@@ -3819,6 +3824,12 @@ UniqueCursorPtr Aggregate::MakeCursor(utils::MemoryResource *mem) const {
   return MakeUniqueCursorPtr<AggregateCursor>(mem, *this, mem);
 }
 
+auto Aggregate::AllAggregationsForCollecting() const -> bool {
```

Comment:
Define the method name like a question to signify from the name that it returns a bool
e.g. `AreAllAggregationsForCollecting`